testing/integration/testdata/templates/filestream-input.tmpl

```diff
@@ -0,0 +1,23 @@
+inputs:
+  - type: filestream
+    id: filestream-e2e
+    use_output: default
+    _runtime_experimental: otel
+    streams:
+      - id: e2e
+        data_stream:
+          dataset: e2e
+        paths:
+          - {{.InputPath}}
+        prospector.scanner.fingerprint.enabled: false
+        file_identity.native: ~
+outputs:
+  default:
+    type: elasticsearch
+    hosts: [{{.ESEndpoint}}]
+    api_key: "{{.ESApiKey}}"
+    preset: "balanced"
+agent:
+  monitoring:
+    metrics: false
+    logs: false
```

Comment:
This test is failing on CI, the config should be:

```yaml
inputs:
  - type: filestream
    id: filestream-e2e
    use_output: default
    _runtime_experimental: otel
    streams:
      - id: e2e
        data_stream:
          dataset: e2e
        paths:
          - {{.InputPath}}
        prospector.scanner.fingerprint.enabled: false
        file_identity.native: ~
outputs:
  default:
    type: elasticsearch
    hosts: [{{.ESEndpoint}}]
    api_key: "{{.ESApiKey}}"
    preset: "balanced"
  monitoring:
    type: elasticsearch
    hosts: [{{.ESEndpoint}}]
    api_key: "{{.ESApiKey}}"
    preset: "balanced"
agent:
  monitoring:
    metrics: true
    logs: false
    use_output: monitoring
`
```

Reply:
What I was working on is just https://github.com/elastic/elastic-agent/pull/8031. That change lets us use the same agent config for beats processes and beats receivers, and just change the `_runtime_experimental` setting, but I'm not actively working on doing that. We should do it before moving the strings to files though.